types/LinkExitTypes.d.ts

```diff
@@ -0,0 +1,157 @@
+
+    // BANK_TRANSFER_ERROR
+    BANK_TRANSFER_LIMIT_EXCEEDED = "BANK_TRANSFER_LIMIT_EXCEEDED",
+    BANK_TRANSFER_MISSING_ORIGINATION_ACCOUNT = "BANK_TRANSFER_MISSING_ORIGINATION_ACCOUNT",
+    BANK_TRANSFER_INVALID_ORIGINATION_ACCOUNT = "BANK_TRANSFER_INVALID_ORIGINATION_ACCOUNT",
+    BANK_TRANSFER_ACCOUNT_BLOCKED = "BANK_TRANSFER_ACCOUNT_BLOCKED",
+    BANK_TRANSFER_INSUFFICIENT_FUNDS = "BANK_TRANSFER_INSUFFICIENT_FUNDS",
+    BANK_TRANSFER_NOT_CANCELLABLE = "BANK_TRANSFER_NOT_CANCELLABLE",
+    BANK_TRANSFER_UNSUPPORTED_ACCOUNT_TYPE = "BANK_TRANSFER_UNSUPPORTED_ACCOUNT_TYPE",
+    BANK_TRANSFER_UNSUPPORTED_ENVIRONMENT = "BANK_TRANSFER_UNSUPPORTED_ENVIRONMENT",
+
+    // SANDBOX_ERROR
+    SANDBOX_PRODUCT_NOT_ENABLED = "SANDBOX_PRODUCT_NOT_ENABLED",
+    SANDBOX_WEBHOOK_INVALID = "SANDBOX_WEBHOOK_INVALID",
+    SANDBOX_BANK_TRANSFER_EVENT_TRANSITION_INVALID = "SANDBOX_BANK_TRANSFER_EVENT_TRANSITION_INVALID",
+
+    // INVALID_REQUEST
+    MISSING_FIELDS = "MISSING_FIELDS",
+    UNKNOWN_FIELDS = "UNKNOWN_FIELDS",
+    INVALID_FIELD = "INVALID_FIELD",
+    INCOMPATIBLE_API_VERSION = "INCOMPATIBLE_API_VERSION",
+    INVALID_BODY = "INVALID_BODY",
+    INVALID_HEADERS = "INVALID_HEADERS",
+    NOT_FOUND = "NOT_FOUND",
+    NO_LONGER_AVAILABLE = "NO_LONGER_AVAILABLE",
+    SANDBOX_ONLY = "SANDBOX_ONLY",
+    INVALID_ACCOUNT_NUMBER = "INVALID_ACCOUNT_NUMBER",
+
+    // INVALID_INPUT
+    // From above ITEM_LOGIN_REQUIRED = "INVALID_CREDENTIALS",
+    INCORRECT_DEPOSIT_AMOUNTS = "INCORRECT_DEPOSIT_AMOUNTS",
+    UNAUTHORIZED_ENVIRONMENT = "UNAUTHORIZED_ENVIRONMENT",
+    INVALID_PRODUCT = "INVALID_PRODUCT",
+    UNAUTHORIZED_ROUTE_ACCESS = "UNAUTHORIZED_ROUTE_ACCESS",
+    DIRECT_INTEGRATION_NOT_ENABLED = "DIRECT_INTEGRATION_NOT_ENABLED",
+    INVALID_API_KEYS = "INVALID_API_KEYS",
+    INVALID_ACCESS_TOKEN = "INVALID_ACCESS_TOKEN",
+    INVALID_PUBLIC_TOKEN = "INVALID_PUBLIC_TOKEN",
+    INVALID_LINK_TOKEN = "INVALID_LINK_TOKEN",
+    INVALID_PROCESSOR_TOKEN = "INVALID_PROCESSOR_TOKEN",
+    INVALID_AUDIT_COPY_TOKEN = "INVALID_AUDIT_COPY_TOKEN",
+    INVALID_ACCOUNT_ID = "INVALID_ACCOUNT_ID",
+
+    // INVALID_RESULT
+    PLAID_DIRECT_ITEM_IMPORT_RETURNED_INVALID_MFA = "PLAID_DIRECT_ITEM_IMPORT_RETURNED_INVALID_MFA",
+
+    // RATE_LIMIT_EXCEEDED
+    ACCOUNTS_LIMIT = "ACCOUNTS_LIMIT",
+    ADDITION_LIMIT = "ADDITION_LIMIT",
+    AUTH_LIMIT = "AUTH_LIMIT",
+    BALANCE_LIMIT = "BALANCE_LIMIT",
+    IDENTITY_LIMIT = "IDENTITY_LIMIT",
+    ITEM_GET_LIMIT = "ITEM_GET_LIMIT",
+    RATE_LIMIT = "RATE_LIMIT",
+    TRANSACTIONS_LIMIT = "TRANSACTIONS_LIMIT",
+
+    // RECAPTCHA_ERROR
+    RECAPTCHA_REQUIRED = "RECAPTCHA_REQUIRED",
+    RECAPTCHA_BAD = "RECAPTCHA_BAD",
+
+    // OAUTH_ERROR
+    INCORRECT_OAUTH_NONCE = "INCORRECT_OAUTH_NONCE",
+    OAUTH_STATE_ID_ALREADY_PROCESSED = "OAUTH_STATE_ID_ALREADY_PROCESSED",
+}
+
+export enum LinkErrorType {
+    BANK_TRANSFER = 'BANK_TRANSFER_ERROR',
```

Comment:
`Bank_Transfer` doesn't exist on Android

Reply:
Yep, I found these in the backend code.  We may need to follow up with changes on Android.